.erda/migrations/kratos/20210914-kratos-uc-id.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `kratos_uc_userid_mapping` (
+  `id` varchar(50) NOT NULL COMMENT 'uc userid',
```

Comment:
change `id` to `uc_user_id`.

add index for both `uc_user_id` and `user_id` separately.

Reply:
add user_id index